src/cli/main.cpp

```diff
@@ -95,18 +95,37 @@ int main(int argc, char* argv[]) {
 
     auto* const experimentalApp = app.add_subcommand("labs");
 
-    experimentalApp->add_flag_function(
+    auto* const verboseOpt = experimentalApp->add_flag_function(
       "--verbose",
       [](auto count) {
         if (count == 1) {
-          fmt::print("Setting log Level to Debug\n");
+          fmt::print("Setting Log Level to Debug ({})\n", LogLevel::Debug);
           openstudio::Logger::instance().standardOutLogger().setLogLevel(LogLevel::Debug);
         } else if (count == 2) {
-          fmt::print("Setting log Level to Trace\n");
+          fmt::print("Setting Log Level to Trace ({})\n", LogLevel::Trace);
           openstudio::Logger::instance().standardOutLogger().setLogLevel(LogLevel::Trace);
         }
       },
-      "Print the full log to STDOUT");
+      "Print the full log to STDOUT - sets verbosity to Debug if given once and Trace if given twice.");
```

Comment:
I think we should just remove the `--verbose` option... instead of having to maintain two excluding options/flags that ultimately achieve the same goal.

@kbenne thoughts please?